firebase-database/src/main/java/com/google/firebase/database/Query.java

```diff
@@ -278,6 +279,92 @@ public void run() {
       });
   }*/
 
+  /**
+   * Create a query constrained to only return child nodes with a value greater than the given
+   * value, using the given orderBy directive or priority as default.
+   *
+   * @param value The value to start at, inclusive
```

Comment:
s/inclusive/exclusive (and maybe s/at/after, but then I am no longer sure whether this is inclusive or exclusive)